benchmark/common/contract_codes.go

```diff
@@ -0,0 +1,81 @@
+//nolint:lll
+package common
```

Comment:
A suggestion, maybe it will be easier, without the need to have byte code hardcoded. 
We already have some test contracts we use in `e2e` tests, UTs, and other forms of testing. They are in `consensus/polybft/contractsapi/test-contracts` folder. Basically what we do when we need a test contract:

1. Add a test sol contract file to `core-contracts` repo, and compile the contracts.
2. Hardhat generates a `json` file (which is basically the artifact of the given contract) in `core-contracts/artifacts` folder.
3. Take that `json` file and copy it to the `consensus/polybft/contractsapi/test-contracts` folder.
4. In `consensus/polybft/contractsapi/init.go` you can add you test contracts by looking an example of `TestRewardToken`.
5. Wherever you use these `ContByteCode` fields, just reference your `TestContract (*artifact.Artifact).ByteCode`.

I think its much easier to modify these test contracts in the future if we need to, plus the artifact `json` file can be quite useful in debugging, if something goes wrong in the benchmark tests.